staticcheck.conf

```diff
@@ -0,0 +1,2 @@
+# See https://staticcheck.dev/docs/configuration/options/#checks.
+checks = ["all", "-ST1000", "-ST1003", "-ST1005", "-ST1012", "-ST1016", "-ST1020", "-ST1021", "-ST1022"]
```

Comment:
Can we add comments saying what the dropped checks are and why we are dropping them?  I think some of them are dropped to match the default value, but some are new.